### PR TITLE
Fix travis

### PR DESCRIPTION
**Pull request by IvarsKarpics**

### CODE REVIEW

File: test/pytest/test_mxcube_qt.py [hunk 1]
```diff
@@ -4,22 +4,26 @@
...
 
 print("MXCuBE root: %s" % MXCUBE_ROOT)
 
 sys.path.insert(0, MXCUBE_ROOT)
 MXCUBE_GUI_FILE = os.path.join(MXCUBE_ROOT,
                                "configuration/example_mxcube_gui.yml")
-hr_server = os.path.join(MXCUBE_ROOT,
-                         "HardwareRepository/configuration/xml-qt")
-os.environ['HARDWARE_REPOSITORY_SERVER'] = hr_server
+HWR_CONIG = "%s%s%s" % (
```
Comment (rhfogh) [inline test/pytest/test_mxcube_qt.py]: Should that have been 'HWR_CONFIG'?

File: gui/widgets/task_toolbox_widget.py [hunk 2]
```diff
@@ -453,10 +453,7 @@ def collect_now_button_click(self):
         will_mount_sample = False
 
         for item in selected_items:
-            if isinstance(item, (
-                queue_item.SampleQueueItem,
-                queue_item.OpticalCentringQueueItem,
-                queue_item.XrayCenteringQueueItem)):
+            if isinstance(item, queue_item.SampleQueueItem):
```
Comment (rhfogh) [inline gui/widgets/task_toolbox_widget.py]: Does that not undo a very recent fix of yours?

**Review by rhfogh — APPROVED** [review]
I cannot really judge, but all fine by me (assuming you feel OK with the stuff I commented on)

File: gui/widgets/task_toolbox_widget.py (hunk 2 above)
Comment (IvarsKarpics) [reply gui/widgets/task_toolbox_widget.py]: ups... I will revert it.

**Comment by IvarsKarpics** [comment]
Hi @rhfogh 

Is the PR ok now? Can you please merge it?

**Pull request merged**